org.palladiosimulator.analyzer.slingshot.behavior.spd/src/org/palladiosimulator/analyzer/slingshot/behavior/spd/interpreter/entity/trigger/AbstractManagedElementTriggerChecker.java

```diff
@@ -101,7 +106,6 @@ protected FilterResult getResult(final DESEvent event) {
 		}
```

Comment:
The disregard Reason in Line 105 is very misleading, at least it mislead me just now. 

Explanation: With the EMA aggregation, there are two possible reasons for a disregard (not enough values and emit intervall has not passed) but the diregard always claims that there are not enough values, eventhough there *are* enough values, and the actual reason for not emitting an aggregated value is the emit duration. 

Suggestion: Change the disregard reason to something more general, e.g. "Values not emittable." or something like that?